kzg_prover/src/circuits/utils.rs

```diff
@@ -145,10 +144,33 @@ pub fn open_grand_sums(
 ) -> Vec<u8> {
     let challenge = Fp::zero();
     create_opening_proof_at_challenge::<
-        KZGCommitmentScheme<Bn256>,
+        _,
         ProverSHPLONK<'_, Bn256>,
-        Challenge255<G1Affine>,
-        Blake2bWrite<Vec<u8>, G1Affine, Challenge255<G1Affine>>,
+        _,
+        Keccak256Transcript<G1Affine, Vec<u8>>,
+    >(
+        params,
+        &advice_polys[balance_column_range],
+        advice_blinds,
+        challenge,
+        constant_terms,
+    )
+}
+
+// This function for comparison with open_grand_sums, which is used ProverSHPLONK
+pub fn open_grand_sums_gwc(
```

Comment:
Is GWC performing better than SHPLONK?

Reply:
Yes, I updated the result in here: https://hackmd.io/yY8N2nd6RT-giGuGlcrdSA#Benchmark-with-ProverGWC-multi-open-scheme
But we have to benchmark with more than ten currencies. because It has better (or similar performance) in generating proof from UnivarientGrandSum circuit on SHPLONK.